### ISSUE TITLE
Trying to get property 'firstChild' of non-object

### ISSUE THREAD

**Issue by p24-max** [outeredge/magento-structured-data-module#32]
With module installed and enabled, I get this error into the Log when trying to access product listing page.
Product listing page shows no products anymore.

Magento 2.4.3-p1
PHP 7.4

```
[2022-09-02 06:40:29] main.CRITICAL: Notice: Trying to get property 'firstChild' of non-object in /var/www/bos/deployments/3bc27a9477fbcca980060ffd3046bd9b89769f54/vendor/outeredge/magento-structured-data-module/Plugin/Review/Block/Product/ReviewRenderer.php on line 20 {"exception":"[object] (Exception(code: 0): Notice: Trying to get property 'firstChild' of non-object in /var/www/bos/deployments/3bc27a9477fbcca980060ffd3046bd9b89769f54/vendor/outeredge/magento-structured-data-module/Plugin/Review/Block/Product/ReviewRenderer.php on line 20 at /var/www/bos/deployments/3bc27a9477fbcca980060ffd3046bd9b89769f54/vendor/magento/framework/App/ErrorHandler.php:61)"} []
```

**Comment by davidwindell** [comment]
@p24-max thanks for the report, we'll take a look at this

**Comment by davidwindell** [comment]
@p24-max this has been fixed in 3.2.5, thanks for bringing it to our attention
